readme.md

```diff
@@ -103,11 +112,25 @@ Write one Express route for each of four HTTP methods.
 Then, make each route respond with a one-word string containing the RESTful action that would most likely be associated with this route.
 
 ```js
+
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+app.get("/", function(req, res){
+  res.send("Read");
+});
 
+app.post("/", function(req, res){
+  res.send("Create");
+});
+
+app.put("/", function(req, res){
+  res.send("Update");
+});
+
+app.delete("/", function(req, res){
+  res.send("Destroy");
+});
```

Comment:
Overall, this is good, but note that your one-responses are the CRUD actions. CRUD actions are general things that can be done to any data. RESTful actions correspond (mostly) with CRUD, but they're not quite the same thing:

Here's the restful actions for each verb you used above:
* GET -> Show or Index (technically New and Edit use a GET verb as well, since they are just returning a form)
* POST -> Create
* PUT -> Update
* Delete -> Destroy